CMakeLists.txt

```diff
@@ -148,6 +149,7 @@ if(${CMAKE_C_COMPILER} MATCHES "icc" OR ${CMAKE_C_COMPILER} MATCHES "icpc" OR ${
         if(BASEARCH_X86_FOUND)
             set(AVX2FLAG "-mavx2")
             set(SSE2FLAG "-msse2")
+            set(SSE3FLAG "-msse3")
```

Comment:
Shouldn't this be ssse3?

Reply:
Good catch. I must have copied the wrong line. I have fixed it.